src/schema/__tests__/city.test.ts

```diff
@@ -150,6 +150,20 @@ describe("City", () => {
       expect(gravityOptions).toMatchObject({ displayable: true })
       expect(gravityOptions).not.toHaveProperty("discoverable")
     })
+    it("requests shows with location, by default", async () => {
+      await runQuery(query, context)
+      const gravityOptions = context.showsWithHeadersLoader.mock.calls[0][0]
+
+      expect(gravityOptions).toMatchObject({ has_location: true })
+      expect(gravityOptions).not.toHaveProperty("discoverable")
```

Comment:
Let's delete the assertions L158 here and L165 below, since they are not particularly relevant to these two tests.

```suggestion
```